### PR TITLE
Moco

### PR DESCRIPTION
**Pull request by MangoFF**

### CODE REVIEW

File: passl/data/preprocess/basic_transforms.py [hunk 1]
```diff
@@ -941,3 +942,21 @@ def __call__(self, img):
             else:
                 img = ImageOps.solarize(img)
         return img
+
+class GaussianBlur(object):
```
Comment (GuoxiaWang) [inline passl/data/preprocess/basic_transforms.py:946]: 这个已经有了把？

File: passl/scheduler/lr_scheduler.py [hunk 2]
```diff
@@ -200,7 +200,58 @@ def get_lr(self):
...
+    def get_lr(self):
+        for i in range(len(self.milestones)):
+            if self.last_epoch < self.milestones[i]:
+                return self.base_lr * (self.gamma**i)
+        return self.base_lr * (self.gamma ** len(self.milestones))
 
-class MultiStepDecay(lr.MultiStepDecay):
-    def __init__(self, learning_rate, milestones, gamma, last_epoch, **kwargs):
-         super().__init__(learning_rate, milestones, gamma, last_epoch)
+class CosineDecay(lr.LRScheduler):
```
Comment (GuoxiaWang) [inline passl/scheduler/lr_scheduler.py:226]: 这个 `CosineDecay` 是不能用复用TimmCosine吗？
